Cargo.toml

```diff
@@ -13,17 +13,18 @@ edition = "2018"
 readme = "README.md"
 
 [features]
-default = []
+default = ["tokio-rustls"]
```

Comment:
This shouldn't be the default :) By default no specific futures runtime should be preferred.

Reply:
Ahh, yes of course! Went a bit too fast on the push. 
I added it because i cannot use cargo patch with new features.
Will remove again right away.